chapters/VK_KHR_swapchain/wsi.txt

```diff
@@ -660,13 +660,55 @@ include::../../validity/protos/vkAcquireNextImageKHR.txt[]
 When successful, fname:vkAcquireNextImageKHR acquires a presentable image
 from pname:swapchain that an application can: use, and sets
 pname:pImageIndex to the index of that image within the swapchain.
-The presentation engine may: not have finished reading from the image at the
-time it is acquired, so the application must: use pname:semaphore and/or
-pname:fence to ensure that the image layout and contents are not modified
-until the presentation engine reads have completed.
 The order in which images are acquired is implementation-dependent, and may:
 be different than the order the images were presented.
 
+fname:vkAcquireNextImageKHR is similar to a <<devsandqueues-submission,queue
+submission command>>, but it does not require a slink:VkQueue as a
+parameter.
```

Comment:
I should maybe further emphasize this point to be safe.

This effectively means there must be a distinct semaphore provided each frame to avoid double-signal of the same semaphore. I.e. unlike other batches this semaphore signal does not (assumably) cover everything before in submission order, and even if it did we do not even know which queue that will be done on.